README.md

```diff
@@ -67,6 +67,14 @@ Branch | Environment
 `perftest` | `perftest`
 PR branch| `preview`
 
+#### Running tests through Azure Front Door
+
+If you want tests in AAT / Stg environments to run via Azure Front Door, you must add configuration for your application to front door. Have a look at the [HMCTS Way](https://hmcts.github.io/cloud-native-platform/path-to-live/front-door.html#front-door-configuration).
+
+Add a CNAME for your application that points to front door to [azure-private-dns](https://github.com/hmcts/azure-private-dns). See [example](https://github.com/hmcts/azure-private-dns/blob/a79d79eb8050440053b45f1a9663db85badf4973/environments/staging/aat-platform-hmcts-net.yml#L65-L67).
```

Comment:
not a great example as it should be a staging record (include -staging at the end I think it is), this might confuse people